SwanContents/swancontents/serverextension/__init__.py

```diff
@@ -0,0 +1,33 @@
+
+from jupyter_server.utils import url_path_join
+from jupyter_server.base.handlers import path_regex
+from jupyter_server import DEFAULT_TEMPLATE_PATH_LIST
+from jupyter_server.serverapp import ServerApp
+
+from swancontents.serverextension.handlers.download import DownloadHandler
+from swancontents.filemanager.handlers import FetchHandler, ContentsHandler
+import datetime
+import os
+
+
+TEMPLATE_PATH = [os.path.join(os.path.dirname(__file__), "templates")]
+
+# Automatically configure our templates
+@property
+def template_file_path(self):
+    return self.extra_template_paths + TEMPLATE_PATH + DEFAULT_TEMPLATE_PATH_LIST
+ServerApp.template_file_path = template_file_path 
+
+def _load_jupyter_server_extension(serverapp):
```

Comment:
"New handlers" is not new anymore, is it? Please some docstring that specifies the handlers that this server extension installs.

Also please at least a paragraph in the commit message that explains this move, as well as the reason to strip out a few of the handlers and where are those going.